rosidl_parser/rosidl_parser/definition.py

```diff
@@ -410,46 +530,49 @@ class Message:
 
     __slots__ = ('structure', 'constants')
 
-    def __init__(self, structure):
+    def __init__(self, structure: Structure):
         """
         Constructor.
 
-        :param Structure structure: the structure of the message
+        :param structure: the structure of the message
         """
         super().__init__()
         assert isinstance(structure, Structure)
         self.structure = structure
-        self.constants = OrderedDict()
+        self.constants = []
 
 
 class Service:
     """A namespaced type containing a request and response message."""
 
-    __slots__ = ('structure_type', 'request_message', 'response_message')
+    __slots__ = ('namespaced_type', 'request_message', 'response_message')
```

Comment:
Similar to the comment above `namespaced_type` better conveys what is contained in this slot.